toolkit/tools/internal/logger/writerhook.go

```diff
@@ -56,6 +58,13 @@ func (h *writerHook) Fire(entry *logrus.Entry) (err error) {
 		return
 	}
 
+	if h.useColors == false {
+		entry.Message = strings.ReplaceAll(entry.Message, "\x1b[31m", "")
+		entry.Message = strings.ReplaceAll(entry.Message, "\x1b[34m", "")
+		entry.Message = strings.ReplaceAll(entry.Message, "\x1b[0m", "")
+		entry.Message = strings.ReplaceAll(entry.Message, "\x1b[32m", "")
```

Comment:
Let's not hard-code every single colour we use. If we ever add another one, we'll need to keep expanding this section (not to mention someone will even need to know this is here). I'd suggest changing it to a regex instead, which will catch all `\x1b[<digits>m` strings. I think `\\x1b\[[0-9]+m` should work, but please double-check in https://regex101.com/ for instance.

Reply:
Yes, sorry, should have mentioned, this is still work in progress. I'm planning to make this into a regex

Reply:
It looks like both logrus and fatith/color have enable/disable flags for colours, could we just use those somehow? Maybe save some pain here? (I'm not 100% sure, since we have two parallel flows of log data...)

Reply:
The disabling of color in both logrus and fatih/color render the output without color, but do not remove color code that has already been applied to a string.
We are experiencing something similar to this, https://github.com/fatih/color/issues/150, and looks like the fatih/color library is not providing a resolution. So, resorting to regex to clean up the string written to logs.
The coloring of log itself is handled by logger.initLogFile() which sets `useColors  = false`
![image](https://github.com/microsoft/CBL-Mariner/assets/58672330/8542ba59-b7b8-447b-b516-230e41725374)
v/s
![image](https://github.com/microsoft/CBL-Mariner/assets/58672330/276d70b5-3bd4-4219-9815-4ac8ae68ff9e)